deploy/charts/harvester/dependency_charts/multus/values.yaml

```diff
@@ -0,0 +1,35 @@
+# Default values for multus cni
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: nfvpe/multus
+  tag: v3.6
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+
+resources:
+  requests:
+    cpu: 100m
+    memory: 50Mi
+  limits:
+    cpu: 100m
+    memory: 50Mi
+
+nodeSelector:
+  kubernetes.io/arch: amd64
+
+tolerations:
+  - operator: Exists
+    effect: NoSchedule
+  # this toleration is to have the daemonset runnable on master nodes
+  # remove it if your masters can't run pods
+  - key: node-role.kubernetes.io/master
+    effect: NoSchedule
+
+affinity: {}
+
+## Specify the cluster type, default to k3s by the harvester
+## with k3s cluster, the minimum version with v1.19.4+k3s1 is required
+cluster:
+  type: k3s
```

Comment:
I think we should make cni-bin-dir & cni-conf_dir configurable. Cluster type is obscure/unclear

Reply:
updated